app/lib/steps/property_section.rb

```diff
@@ -1,6 +1,6 @@
 module Steps
   class PropertySection
-    PROPERTY_STEPS = %i[property property_landlord property_entry].freeze
+    PROPERTY_STEPS = %i[property property_landlord cannot_use_service property_entry].freeze
```

Comment:
we add `cannot_use_service` to `PROPERTY_STEPS` doesn't that mean that we will always see that page when `PROPERTY_STEPS` is called? I don't see in the file where we are determining when to show that page?

Reply:
@willc-work this also really confused me. 

I think this is my understanding: `PropertySection` is only called by `all_sections` in `app/lib/steps/helper.rb`, and it needs it to know of all the sections that we have. The logic of what page to show in the flow, is dictated by `app/lib/steps/outgoings_section.rb` and the logic of the flow is there.